### ISSUE TITLE
chunks folder can't merge into 1 file

### ISSUE THREAD

**Issue by Killoude** [pionl/laravel-chunk-upload-example#13]
hello I was trying to upload the big files about (5M)
and i chunked into 2M. 

when I looked on my upload folder, I only got 1 chunked file(it isn't merged file)
does the folder structure has an effect on it?

my structure folder ( the directory is storage/app/)

1. testfolders (i put tmp,delete folder here)
    testfolders/tmp -> temporary upload
    testfolders/delete -> if user delete the file
2. chunks folders

and the extension is only allow .zip

source is the same as an example code one.

thank you very much

**Comment by Killoude** [comment]
I forgot to copy the source (my mistake)

**Controller** 

```
public function uploadfiles(Request $request)
     {
        $receiver = new FileReceiver("testfolders", $request, HandlerFactory::classFromRequest($request));
    
        if ($receiver->isUploaded()) {
         
        $save = $receiver->receive();
        
        // 分割終了
        if ($save->isFinished()) 
        {
            return $this->saveFile($save->getFile());
        }
        else{
        $handler = $save->handler();
        
        return response()->json([
            "done" => $handler->getPercentageDone(),
            'status' => true
        ]);
        }
        }
        else{
            throw new UploadMissingFileException();
        }
    }

    //ファイル保存
    protected function saveFile(UploadedFile $file)
    {
        $filename = $this->createFilename($file);
        
        // // chunkファイル1つにする
        // $mime = str_replace('/', '-', $file->getMimeType());

        //tmpファイルへ移動
        $temp_path = "testfolders/tmp/";        
        $finalPath = storage_path("app/".$temp_path);
        
        $file->move($finalPath, $filename);
        
        $folder_object = new \stdClass();
        $folder_object->name = $filename; //仮名
        $folder_object->real_name =$file->getClientOriginalName();
        $folder_object->path = $temp_path.'/'.$filename;
        $folder_object->size = round($file->getClientSize() / 1024, 2)/1000000;
        // $folder_object->mime_type=$mime;
        $folders[] = $folder_object;
        return response()->json(array('files' => $folders), 200);
    }

    //ファイル名を作成
    protected function createFilename(UploadedFile $file)
    {
        $extension = $file->getClientOriginalExtension();
        $filename = str_replace(".".$extension, "", $file->getClientOriginalName()); 
        // Add timestamp hash to name of the file
        $filename .= "_" . md5(time()) . "." . $extension;
        return $filename;
    }
```

**.js**
```
$(function () {
    $.ajaxSetup({ headers: { 'X-CSRF-TOKEN': $('meta[name="csrf-token"]').attr('content')}});
    $('#fileupload').fileupload({
        method: 'POST',
        maxFileSize: 10000000000,//10GB
        maxChunkSize: 2000000,//2mb
        dataType: 'json',
        add: function (e, data) {
        $('#loading').text('Uploading...');
        data.submit();
        },
        progress: function (e, data) {
        $('#fileupload').attr('disabled','disabled');     
        var progress = parseInt(data.loaded / data.total * 100, 10);
        $('.progress').css({ 'width': progress + '%' });
        $('#loading').text('アップロード中..'+progress+'%');
        },    
        fail: function (e, data) {
        $('#loading').text('アップロード失敗、サイズをご確認ください(最大10GB)');
        return false;
        },      
        done: function (e, data) {
            console.log(data, e);
            $.each(data.result.files, function (index, file) {
            var removeLink = $('<a href="javascript:;">削除</a>');
            $('<p id="filepoint">').html(file.real_name +'('+file.size+' GB)').appendTo($('#files_list')).append(removeLink);
          
            $('#file_names').val(file.name);
            $('#real_names').val(file.real_name);
            $('#file_paths').val(file.path);
                
            $('#loading').text('');
            removeLink.click(function() {
            return removeFile(file.name); });    
            });
        }
    });
});
```

**Comment by pionl** [comment]
Are you running Laravel 7?

**Comment by pionl** [comment]
Fixed.

🎉 Finally a release for Laravel 7 support (bug found by @drjdr)
⛑ Finally I've fixed problems with chunks (Dropzone/Resumable.js with parallel upload). 
👍 Also I've found a solution how to get file data after upload for a Dropzone. Check [WIKI](https://github.com/pionl/laravel-chunk-upload/wiki/Dropzone#get-final-response-data).

Now the release cycle will be easier thanks to integrations test on all Laravel versions. Update should take less time without breaking any support.
